src/main/java/org/nest/symboltable/predefined/PredefinedFunctions.java

```diff
@@ -134,13 +134,13 @@
     final MethodSymbol i_sum = createFunctionSymbol(CURR_SUM);
     i_sum.addParameterType(getType("pA"));
     i_sum.addParameterType(getBufferType());
-    i_sum.setReturnType(getType("pA"));
+    i_sum.setReturnType(getType("nS"));
     name2FunctionSymbol.put(CURR_SUM, i_sum);
 
     final MethodSymbol cond_sum = createFunctionSymbol(COND_SUM);
     cond_sum.addParameterType(getType("nS"));
     cond_sum.addParameterType(getBufferType());
-    cond_sum.setReturnType(getType("pA"));
+    cond_sum.setReturnType(getType("nS"));
```

Comment:
Which models are concerned? After my PR, the ``aeif_cond_alpha`` and ``aeif_cond_exp`` do not work because of the ``min`` function which breaks ``sympy``, so the fact that these models donot work is "normal" (read "needs fixing" ^^)